pillowtop/listener.py

```diff
@@ -230,11 +207,11 @@ def change_trigger(self, changes_dict):
         if self.use_locking:
             lock = self.document_class.get_obj_lock_by_id(id)
             lock.acquire()
-            return LockManager(self.couch_db.open_doc(id), lock)
+            return LockManager(self._couch_db.open_doc(id), lock)
         elif changes_dict.get('doc', None) is not None:
             return changes_dict['doc']
         else:
-            return self.couch_db.open_doc(id)
+            return self._couch_db.open_doc(id)
```

Comment:
makes sense

On 8 October 2015 at 13:38, Cory Zue <notifications@github.com> wrote:

> In pillowtop/listener.py
> <https://github.com/dimagi/pillowtop/pull/154#discussion_r41502813>:
>
> >          elif changes_dict.get('doc', None) is not None:
> >              return changes_dict['doc']
> >          else:
> > -            return self.couch_db.open_doc(id)
> > +            return self._couch_db.open_doc(id)
>
> because i want to eventually soft_assert that nothing uses the public
> version and everything funnels through a DocumentStore object
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/pillowtop/pull/154/files#r41502813>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa